ugc/storage.cpp

```diff
@@ -72,6 +73,62 @@ string SerializeUGCIndex(vector<Storage::UGCIndex> const & indexes)
   unique_ptr<char, JSONFreeDeleter> buffer(json_dumps(array.get(), JSON_COMPACT | JSON_ENSURE_ASCII));
   return string(buffer.get());
 }
+
+template <typename UGCUpdate>
+Storage::SettingResult SetGenericUGCUpdate(
+    vector<Storage::UGCIndex> & indexes, size_t & numberOfDeleted, FeatureID const & id,
+    UGCUpdate const & ugc,
+    function<unique_ptr<FeatureType>(FeatureID const &)> const & getFeatureFn,
```

Comment:
Why not to pass FeatureType explicitly??? You use this stange beast only once, to get feature type, right?

Reply:
Moreover, Feature Id is stored in FeatureType.

Reply:
Yes, it was a strange decision. I replaced it with an explicit ```GetFeature``` call.